docs/concepts/07_dip/05_dapp_developer.md

```diff
@@ -26,8 +26,128 @@ Import the SDK into your code:
 import { * } from '@kiltprotocol/dip-sdk'
 ```
 
+## Example application
+
+This example application walks through the code you can find in the tests for the DIP SDK.
+
+### 1. Generate a base proof
+
+Start by generating the base proof for the DIP using the `[generateDipSiblingBaseProof](https://kiltprotocol.github.io/dip-sdk/functions/generateDipSiblingBaseProof.html)` method and passing the desired configuration:
+
+```typescript
+const baseDipProof = await DipSdk.generateDipSiblingBaseProof(config)
+```
+
+:::info What's a base proof?
+A base proof is a cross-chain state proof, revealing the parts of a DID stored on the KILT blockchain.
```

Comment:
So, technically a DID is just an identifier, while the DID Document is the set of keys associated with it. I'm not suggesting it must be changed here, but it's probably good for me to share with you this is the framework that we should be thinking in, according to the spec. But I've seen plenty of time DID being used in place of DID Document.

```suggestion
A base proof is a cross-chain state proof, revealing the parts of a DID Document stored on the KILT blockchain.
```